src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double width;
+    private double height;
+
+    public Rectangle(String color, double width, double height) {
+        super(color);
+        this.width = width;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return width * height;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure rectangle, area: " + getArea() + " width" + width
+                + " ,height" + height + " color " + color);

```

Comment:
There is a missing space after 'width' and before 'height' in the `System.out.println()` statement. Consider adding spaces to improve the readability of the output.